src/TgSharp.Core/Network/MtProtoPlainSender.cs

```diff
@@ -7,21 +7,38 @@ namespace TgSharp.Core.Network
 {
     public class MtProtoPlainSender
     {
+        public delegate void ResponseHandler(byte[] response);
```

Comment:
delegates are old school, just use Funcs or Actions

Reply:
should be fixed